### ISSUE TITLE
[주간회의] 10월 26일

### ISSUE THREAD

**Issue by kangdain** [kangdain/youtuboard#13]

**Comment by kangdain** [comment]
### 한 일


-   1. 제 6 차 TLO 교육 : 4차 산업혁명 시대에는 어떤 인재가 필요한가 ?
- 2. ReLU and 초기값 정하기 -  NN에서 Xavier초기화를 통해 초기 cost값을 낮게 출력되어지도록 만들어 준뒤, 정확도를 조금 더 높혀주기 위한 방법으로 Deep Neural Network 를 사용할 수 있는데, 이 경우 Overfitting이 발생할 수 있기 때문에 Dropout 으로 이를 해결 할 수 있다. (월) 
-  CNN introduction: CNN 에서 convolution layer를 sampling 하여 새로운 레이어를 얻는 pooling 과 처음부터 끝까지 하나로 연결되어있는 layer(Fully connected layer )의 개념 및 실습을 통해 CNN 이 이미지를 판단하는 과정에 대해 알수 있었다. (화)
- Vue Router 에 대한 학습 - Vue.router 를 설치하여 컴포넌트 이동에서 history 모드로 이동하는 방법 및 child routes를 만들어 중첩된 구조를 형성할 수 있다. (수)
 
- 쿼리 파라미터 & 와일드 카드 - 사용자가 URL로 접근하려고 할때 쿼리가가 있는 경우$route.query 로 접근 가능하며, 존재 하지않는 URL일 경우 와일드 카드로 홈으로  Redirect 시켜줄있다. (목)
- Navigation guard - 네비게이션 가드를 통해 사용자가 특정 URL 진입 시 해당 URL을 beforeEnter 와 beforeLeave 진입을 통제할 수 있다. (금)
 
### 할일 
월화 - Navigation.vue 마무리 , 라우터 설정
목 - 스터디 시작
